docs/single-spa-api.md

```diff
@@ -1,19 +1,13 @@
 # single-spa API
 The single-spa library does not `export default`, but instead exports named functions and variables.
 
-## setLoader
-`setLoader(Loader)` sets the javascript [loader](https://whatwg.github.io/loader/) that will be used by single-spa.
-A loader must implement `Loader.import(...).then(...).catch(...)`, and the most commonly used loader is
-[SystemJS](https://github.com/systemjs/systemjs). This API should be called **before** any `declareChildApplication`
-calls are made.
-
 ## start
 `start()` is a function that must be called by your root application. Before `start` is called, child
 applications will be loaded, but will never be bootstrapped, mounted or unmounted.
```

Comment:
Would it be worth adding a sentence as to why `start` exists? What have it explicit over implied?